pkg/query/querier_test.go

```diff
@@ -848,16 +793,9 @@ func TestSortReplicaLabel(t *testing.T) {
 	}
 }
 
-const hackyStaleMarker = float64(-99999999)
-
 func expandSeries(t testing.TB, it chunkenc.Iterator) (res []sample) {
 	for it.Next() {
 		t, v := it.At()
-		// Nan != Nan, so substitute for another value.
```

Comment:
Let's leave it - otherwise we will have nondeterministic result someday as Nans are tricky. What's the blocker for leaving those? (: